sdk/go/auto/stack.go

```diff
@@ -290,7 +290,7 @@ func (s *Stack) Preview(ctx context.Context, opts ...optpreview.Option) (Preview
 	// Apply the remote args, if needed.
 	sharedArgs = append(sharedArgs, s.remoteArgs()...)
 
-	kind, args := constant.ExecKindAutoLocal, []string{"preview"}
+	kind, args := constant.ExecKindAutoLocal, previewOptsToCmd(preOpts, s)
```

Comment:
Why this `previewOptsToCmd`?  We're already appending the shared args above (in fact it looks like `--config-file` is already appended to the args.  If that doesn't work, I don't think this would either?  Also this would mean we don't append the args when there's an inline program.